src/main.rs

```diff
@@ -0,0 +1,58 @@
+use clap::{arg, value_parser, Command};
+
+fn cli() -> Command {
+    Command::new("state-reconstruct")
+        .about("zkSync state reconstruction tool")
+        .subcommand_required(true)
+        .arg_required_else_help(false)
+        .subcommand(
+            Command::new("import")
+                .about("Import state")
+                .subcommand_required(true)
+                .subcommand(
+                    Command::new("l1")
+                        .about("Import state from Ethereum L1")
+                        .arg(
+                            arg!(--"start-block" <START_BLOCK>)
+                                .help("Ethereum block number to start state import from")
+                                .default_value("16627460")
+                                .value_parser(value_parser!(u64)),
+                        )
+                        .arg(
+                            arg!(--"block-step" <BLOCK_STEP>)
+                                .help("Number of blocks to filter & process in one step")
+                                .default_value("128")
+                                .value_parser(value_parser!(u64)),
+                        ),
+                )
+                .subcommand(
+                    Command::new("file")
+                        .about("Import state from file")
+                        .arg(arg!(<FILE> "File to import state from"))
+                        .arg_required_else_help(true),
+                ),
+        )
+}
+
+fn main() {
+    let matches = cli().get_matches();
+
+    match matches.subcommand() {
+        Some(("import", sub_matches)) => {
```

Comment:
```suggestion
        Some((Command::Reconstruct, sub_matches)) => {
```
Do you know if there's any way to have the commands and sub-commands as `Enum`s, like we can with the `#[derive(Parser)]` method? Would make it easier to maintain in the future.

Reply:
I briefly looked at this while writing the code. You can create an enum and workaround it here, but I personally would lean towards not having that boilerplate since I don't think these few string occurrences can cause any issues here.